definition/rooms/IRoomUserTyping.d.ts

```diff
@@ -9,5 +9,5 @@ import { IRoomUserTypingContext } from './IRoomUserTypingContext';
  *
  */
 export interface IRoomUserTyping {
-    executeOnRoomUserTyping(context: IRoomUserTypingContext, read: IRead, http: IHttp, persistence: IPersistence): Promise<void>;
+    executeOnRoomUserTyping(context: IRoomUserTypingContext, read: IRead, http: IHttp, persistence: IPersistence, modify: IModify): Promise<void>;
```

Comment:
@csmith14 We can't directly update files in `definition/`. these are build files.

You can modify it in the `src` folder and after that `yarn compile` will generate all the build files